backend/src/main/kotlin/org/loculus/backend/model/SubmitModel.kt

```diff
@@ -313,4 +316,13 @@ class SubmitModel(
             throw UnprocessableEntityException(metadataNotPresentErrorText + sequenceNotPresentErrorText)
         }
     }
+
+    @Transactional(readOnly = true)
+    fun checkIfStillProcessingSubmittedData(): Boolean {
```

Comment:
We could eventually allow filtering on the submitting group rather than on any entries as only the same group matters for ingest purposes, but this shouldn't be an issue unless we always have something in aux table which is unlikely to ever be the case.

Reply:
If you think it's necessary, create an issue. :) But it's definitely out of scope for this PR.